config/theme-tabler.php

```diff
@@ -19,22 +19,10 @@
      * Simple pick a layout and let Backpack decide the best look for it.
      * No extra step is required.
      *
-     * Where should the menu be positioned?
+     * Possible values: horizontal, horizontal_dark, horizontal_overlap, vertical,
+     * vertical_dark, vertical_transparent, right_vertical, right_vertical_dark, right_vertical_transparent
      */
     'layout' => 'horizontal',
-    // horizontal
-    // horizontal_dark
-    // horizontal_overlap
-    // vertical
-    // vertical_dark
-    // vertical_transparent
```

Comment:
In the `vertical_transparent` layout, because the topbar is present and fixed (aka sticky), when you scroll down the buttons are still visible and overlap with the content. No bueno:

![CleanShot 2023-04-19 at 11 15 03@2x](https://user-images.githubusercontent.com/1032474/233012609-dab7d079-32da-4170-84b5-42deb3d6e2d7.png)

This would also be fixed by removing the topbar from this layout.